src/main/scala/mesosphere/mesos/TaskGroupBuilder.scala

```diff
@@ -157,16 +162,19 @@ object TaskGroupBuilder {
     podDefinition: PodDefinition,
     portsMatch: PortsMatch,
     portMappings: Seq[mesos.NetworkInfo.PortMapping],
-    instanceId: Instance.Id): mesos.ExecutorInfo.Builder = {
+    instanceId: Instance.Id,
+    frameworkId: mesos.FrameworkID): mesos.ExecutorInfo.Builder = {
     // TODO: use only an instance id.
     val executorID = mesos.ExecutorID.newBuilder.setValue(f"marathon-${instanceId.idString}")
 
     val executorInfo = mesos.ExecutorInfo.newBuilder
       .setType(mesos.ExecutorInfo.Type.DEFAULT)
       .setExecutorId(executorID)
+      .setFrameworkId(frameworkId)
 
     executorInfo.addResources(scalarResource("cpus", PodDefinition.DefaultExecutorCpus))
     executorInfo.addResources(scalarResource("mem", PodDefinition.DefaultExecutorMem))
+    executorInfo.addResources(scalarResource("disk", 0))
```

Comment:
if disk space is ever enforced, I wonder how it will be enforced for pods. for example, pod subcontainers can already specify disk resources but we don't add any "default" disk resources for the executor. but there are almost certainly log files that will accumulate in the executor's sandbox. and the executor sandbox will also hold all of the subcontainer sandboxes .. hopefully those won't count against the pod executor's disk resource allocation.